### PR TITLE
First pass in the port, code cleanup

### PR DESCRIPTION
**Pull request by AndrewFalanga**
This part is just a bunch of code cleanup. The porting to UDP from TCP will come in a bit.

### CODE REVIEW

**Review by rfalanga — COMMENTED** [review]
It looks OK to me, but I'm wondering what your motivation is on line 102 and 145 in tss.c? The changes that you're making there. If I've correctly interpreted what the `git diff` is telling me.

**Comment by AndrewFalanga** [comment]
> It looks OK to me, but I'm wondering what your motivation is on line 102 and 145 in tss.c? The changes that you're making there. If I've correctly interpreted what the `git diff` is telling me.

I'm not sure I found what you meant. Can you put a comment directly on that line?

**Comment by rfalanga** [comment]
Unfortunately, no I can't. Code reviewing in GitHub is still very much unfamiliar territory for me.

**Comment by AndrewFalanga** [comment]
> Unfortunately, no I can't. Code reviewing in GitHub is still very much unfamiliar territory for me. 

It is kinda strange. I like it less than BitBucket.

File: src/overwrite.c
Comment (rfalanga) [inline src/overwrite.c:1]: Looks like you're consolidating this code into tss_common.c

**Review by rfalanga — APPROVED** [review]

**Comment by rfalanga** [comment]
Andy,

Thank you so much for this exercise! This is the first time, ever, I've done a PR on GitHub (or anyplace else), and so I need this experience!! And at work we're migrating to GitHub, so we should start doing this. Whether we do or not, waits to be seen. Anyway, the important thing is that I'm gaining experience at a common and good practice, regardless of whether to coworkers want to do it or not.


Rod
________________________________

**Comment by AndrewFalanga** [comment]
You’re welcome. I need these interactions too. I plan to challenge another
class at BSU and will need things like this to use in the process. I can’t
share code from work so this is the next best thing

I hope you are able to steer them toward PRs. They really help make things
nice.

**Comment by rfalanga** [comment]
No problem, Andy. I also cannot share code from work, so I get it. But PRs is so important! I will create a PR on code I work on, just to sort of practice. Although, it's like pretending that Santa Claus is about to come down the stairs, but I'm Santa. I've only participated three times with someone else, on a PR, coincidentally on GitHub. One was on an open-source project I did a bit with. My PR was accepted, but it was very small. Another was with the .NET user group in Albuquerque, which I did years ago, and no one has responded to it. And the third was your PR to me on cleaning up the code.

I hope that I've done the right thing and that the PR is closed and accepted by me.


Rod

**Pull request merged**